pages/common/astyle.md

```diff
@@ -0,0 +1,24 @@
+# astyle
+
+> Source code indenter, formatter, and beautifier for the C, C++, C# and Java programming languages.
+> A copy of the original file is created with an dot orig appended to the original file name.
```

Comment:
I dont understand this line. What does it mean ? And why is it needed ?

Reply:
When you run astyle it formats your file and creates a backup of the original file.
 
When you run  ```astyle myfile.java```

The output will be  ```myfile.java and myfile.java.org```